src/util/get-property-descriptor.js

```diff
@@ -3,7 +3,7 @@ const div = document.createElement('div');
 function getPrototype(obj, key) {
   let descriptor;
 
-  while (obj && !(descriptor = Object.getOwnPropertyDescriptor(obj, key))) {
+  while (obj && !(descriptor = Object.getOwnPropertyDescriptor(obj, key))) { // eslint-disable-line max-len, no-cond-assign
```

Comment:
it seems like the eslint max-line length rule is disabled a few times, is it worth us overriding this rule with a higher value?